src/components/component.ts

```diff
@@ -236,9 +237,19 @@ export class Component {
     if (this._clipPathEnabled) {
       this._updateClipPath();
     }
+    this._renderImmediately();
```

Comment:
the 3 lines above were not being called in `Axis` and its children, and now they are. 
was it a conscious decision?